contributing_to_docs/doc_guidelines.adoc

```diff
@@ -20,6 +20,10 @@ It is not necessary to update existing content to unwrap lines, but you can remo
 In the Atom editor, you can use `Ctrl`+`J` to undo hard wrapping on a paragraph.
 ====
 
+* The following books are auto-generated and should not be manually edited:
+** `cli_reference`
+** `rest-api`
+
```

Comment:
As @abhatt-rh recommended, we could put something afterward here to help people with what to do if something needs updated in either of these files. Maybe something like:

```
If content in these files need to be updated, the update must be made in the applicable code repository where these docs are generated from. The updates will then be reflected the next time the docs are generated; likely a future release. For help with where to make the updates, contact Andrea Hoffer for the CLI docs or Jason Boxman for the API docs.
```
Could maybe even add a link to me and @jboxman-rh's GitHub accounts in there?